### PR TITLE
Deal with form data

### PR DESCRIPTION
**Pull request by tyx**

### CODE REVIEW

File: features/send_request.feature [hunk 1]
```diff
@@ -16,12 +16,22 @@ Feature: Test send API request
         When I send a POST request to "echo" with body:
         """
         {
-            "name" : "name",
-            "pass": "pass"
+            "username" : "pablo",
+            "password": "money"
```
Comment (shouze) [inline features/send_request.feature]: It's honey no?

**Comment by shouze** [comment]
LGTM, checked against a project wich had some problems without this patch :+1:

File: src/Rest/RestApiBrowser.php [hunk 2]
```diff
@@ -116,17 +116,19 @@ public function sendRequest($method, $url, $body = null)
...
      */
     private function send($method, $uri, $body = null)
     {
         if (!$this->hasHost($uri)) {
             $uri = rtrim($this->httpClient->getConfiguration()->getBaseUri(), '/') . '/' . ltrim($uri, '/');
         }
+        $body = is_array($body) ? http_build_query($body) : $body;
         $stream = new Stream('php://memory', 'rw');
-        if ($body) {
+        if (is_string($body)) {
```
Comment (stephpy) [inline src/Rest/RestApiBrowser.php]: is_scalar would be better. Because body could be an int. :)

Comment (shouze) [inline src/Rest/RestApiBrowser.php]: Yup you're right :+1:  so we then cast it explicitely into string in the stream write then.

Comment (stephpy) [inline src/Rest/RestApiBrowser.php]: Not necessary, developer should have this responsability I guess.

**Pull request merged**